frontend/src/components/common/SideTab.tsx

```diff
@@ -1,10 +1,82 @@
-import React from "react";
-
-const SideTab = () => (
-  <>
-    SideTab
-    {/* TODO: SideTabを実装 */}
-  </>
-);
+import { AccountCircle, ListAlt, QrCode } from "@mui/icons-material";
+import {
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from "@mui/material";
+import { useRouter } from "next/router";
+
+const SideTab = () => {
+  const router = useRouter();
+  const handleClick = (link: string) => {
+    router.push(link).catch((error) => {
+      console.error(error);
+    });
+  };
+
+  const tabs = [
+    {
+      key: "home",
+      name: "ホーム",
+      icon: <AccountCircle />,
+      url: "/",
+    },
+    {
+      key: "personal-info",
+      name: "個人情報",
+      icon: <ListAlt />,
+      url: "/personal-info",
+    },
+    {
+      key: "qrcode",
+      name: "学祭QRコード",
+      icon: <QrCode />,
+      url: "/qrcode",
+    },
+  ];
+
+  return (
+    <List sx={{ width: 280 }}>
+      {tabs.map((tab) => (
+        <ListItem
+          key={tab.key}
+          disablePadding
+          sx={{
+            width: 280,
+            height: 44,
+          }}
+        >
+          <ListItemButton
+            autoFocus={router.pathname === tab.url}
+            onClick={() => handleClick(tab.url)}
+            sx={{
+              borderTopRightRadius: 22,
+              borderBottomRightRadius: 22,
```

Comment:
[note] 本家を見に行くと、border-radiusが50pxだったので一応

```suggestion
              borderTopRightRadius: 25,
              borderBottomRightRadius: 25,
```

Reply:
あれこれはまじか
Tabの高さが44と思ったからこう設定したんだけどそもそもTabの高さが間違ってる説あるな